experimental/grouped-and-auto-accessors.md

```diff
@@ -0,0 +1,85 @@
+# [Grouped Accessors and Auto-Accessors][proposal-grouped-and-auto-accessors]
+
+## ClassBody
+
+```js
+extend interface ClassBody {
+    body: [ MethodDefinition | PropertyDefinition | StaticBlock | ClassAccessor ];
+}
+```
+
+## ObjectExpression
+
+```js
+extend interface ObjectExpression {
+    properties: [ Property | ObjectAccessor ];
+}
+```
+
+## ClassAccessor
+
+```js
+interface ClassAccessor <: PropertyDefinition {
+    type: "ClassAccessor";
+    get: GetAccessorStub | GetAccessorMethod | null;
+    set: SetAccessorStub | SetAccessorMethod | null;
+}
+```
+
+If `set.private` is `true`, `key` must be an `Identifier` and `computed` must be `false`.
+
+## ObjectAccessor
+
+```js
+interface ObjectAccessor <: Property {
+    type: "ObjectAccessor";
+    get: GetAccessorStub | GetAccessorMethod | null;
+    set: SetAccessorStub | SetAccessorMethod | null;
+}
+```
+
+### GetAccessorStub
+
+```js
+interface GetAccessorStub <: Node {
+    type: "GetAccessorStub";
+}
+```
+
+A get accessor stub is a simplified version of grouped accessor. E.g. `class C { accessor c { get; } }`.
+
+### GetAccessorMethod
+
+```js
+interface GetAccessorMethod <: Node {
+    type: "GetAccessorMethod";
+    params: [];
```

Comment:
The `params` must be an empty array. It is to provide compatibility for general function AST visitors: A function always has `params` property.